### PR TITLE
[WIKI] Modify Contributing on repo

### PR DESCRIPTION
**Pull request by BlinkCreator**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -24,7 +27,7 @@ Email `se701group1@gmail.com` to ask for access to the prod environment.
 Deployment of the frontend and backend
 
 ### Architecture
-A description of the architecture can be found within the wiki. Changes to the architecture can be 
+A description of the [architecture](https://github.com/we-are-number-1/yumble/wiki/Yumble-Architecture-&-Design) can be found within the wiki. Backlog of aritecture can be found on the groups google-drive.git and must be requested from se701group1@gmail.com
```
Comment (Alxjlv) [inline CONTRIBUTING.md:30]: What do you mean by backlog of architecture? Haha also, what is google-drive.git?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -38,5 +41,4 @@ A description of the architecture can be found within the wiki. Changes to the a
 * Ask `se701group1@gmail.com` to be added as an outside collaborator
```
Comment (Alxjlv) [inline CONTRIBUTING.md:41]:
```suggestion
* Email `se701group1@gmail.com` in order to be added as an outside collaborator
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -4,10 +4,13 @@
...
-* Tickets, pull requests all follow the same schema. 
-* Refer to the [Yumble Wiki]: https://github.com/we-are-number-1/yumble/wiki/Ticket-&-Commit-Schema for futher details on the schema yumble will be using
+* All issue tickets, feature requests and bug reports must be approved through this [process](https://github.com/we-are-number-1/yumble/wiki/Issue-Review-&-Approval-process) before work is completed.
+* Here is an updated list of all [approved labels](https://github.com/we-are-number-1/yumble/wiki/Label-Definitions) for issues.
+
+### Pull Request approval and approval process
+* All pull requests must follow this [process](https://github.com/we-are-number-1/yumble/wiki/Pull-Request-Review-and-Approval-Process)
 
 ### Developer Environment
 * Node 15.x
```
Comment (Alxjlv) [inline CONTRIBUTING.md:16]:
```suggestion
* Node 15.11
```
I think a higher minor version of node broke it

**Review by Alxjlv — CHANGES_REQUESTED** [review]
Just a quick couple of changes!

**Review by David-Xia0 — APPROVED** [review]
Yeah see Alexs comment / change request
but apart from that LGTM

**Review by Alxjlv — APPROVED** [review]
👍

**Review by Jaysce — APPROVED** [review]
LGTM 👍

**Pull request merged**